test/e2e/wait_util.go

```diff
@@ -0,0 +1,57 @@
+package e2e
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	userv1 "github.com/openshift/api/user/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+const (
+	retryInterval        = time.Second * 5
```

Comment:
the retry interval could be shorter (eg 1sec) to make the test as fast as possible.

Reply:
Taken from sample operator_e2e_test from operator_sdk [link](https://github.com/operator-framework/operator-sdk-samples/blob/master/memcached-operator/test/e2e/memcached_test.go#L33).  Hope this make sense.